mungegithub/mungers/submit-queue.go

```diff
@@ -72,7 +72,7 @@ const (
 var (
 	_ = fmt.Print
 	// This MUST cause a RETEST of everything in the sq.RequiredRetestContexts
-	retestBody = fmt.Sprintf("@%s test this [submit-queue is verifying that this PR is safe to merge]", jenkinsBotName)
```

Comment:
noted! i will need to have both new and old retest body. i will make the relevant changes.